css/style.css

```diff
@@ -0,0 +1,124 @@
+
+body {
+  background-color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  padding: 0;
+  margin: 0;
+}
+
+/* HEADER-MOBILE */
+header {
+  padding: 0;
+  margin: 1px;
+  display: flex;
+  width: 100;
+  height: 1fr;
+  background-color: rgb(255, 255, 255);
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+}
+
+#main-container .introduction h1 {
+  font-size: 40px;
+  text-align: center;
+  color: #172b4b;
+  font-weight: 700;
+
```

Comment:
- [ ] As suggested by the previous reviewer [here](https://github.com/SamuelTTW/Portfolio-Samuel-Parra-2/pull/1#discussion_r884677996), kindly add the font family "Inter" for introduction h1.